lib/core/src/Cardano/Wallet/DB/Sqlite/Types.hs

```diff
@@ -394,6 +394,12 @@ instance PathPiece PoolId where
     fromPathPiece = fromTextMaybe
     toPathPiece = toText
 
+instance ToJSON PoolId where
+    toJSON = String . toText
+
+instance FromJSON PoolId where
+    parseJSON = aesonFromText "PoolId"
```

Comment:
:man_shrugging: I still don't understand why persistent requires this for field involved in a PrimaryKey. This is non-sense.